engine/class_modules/sc_rogue.cpp

```diff
@@ -10072,6 +10268,12 @@ void rogue_t::init_spells()
   spell.subterfuge_buff = ( talent.rogue.subterfuge->ok() || talent.outlaw.underhanded_upper_hand->ok() ) ? find_spell( 115192 ) : spell_data_t::not_found();
   spell.vanish_buff = spell.vanish->ok() ? find_spell( 11327 ) : spell_data_t::not_found();
 
+  // Hero Talent Background Spells
+  spell.fatebound_coin_heads_buff = talent.fatebound.hand_of_fate->ok() ? find_spell( 452923 ) : spell_data_t::not_found();
+  spell.fatebound_coin_tails_buff = talent.fatebound.hand_of_fate->ok() ? find_spell( 452917 ) : spell_data_t::not_found();
```

Comment:
Oh, an aside: The spell data for these says they have a 3600 second expiry - an hour. This is not shown in-game, and I have yet to idle with one for an hour to see if it actually disappears. Something to check later, I suppose. It won't matter for any sims - an hour is effectively forever for a sim.